Use PySide6 instead of PySide6-Essentials to be compatible with PySide6
EasyBuild package.

diff --git a/setup.py b/setup.py
index e6e9347..7ca2e9b 100644
--- a/setup.py
+++ b/setup.py
@@ -54,7 +54,7 @@ setup(
     python_requires='>=3.8',
     include_package_data=True,
     install_requires=[
-        "PySide6-Essentials~=6.4",
+        "PySide6~=6.4",
         "numpy~=1.18",
         "matplotlib~=3.6",
         "tqdm~=4.47",
-- 
2.34.1

